python_on_whales/docker_client.py

```diff
@@ -236,6 +236,7 @@ def login_ecr(
         aws_access_key_id: Optional[str] = None,
         aws_secret_access_key: Optional[str] = None,
         region_name: Optional[str] = None,
+        registry: Optional[str] = None
```

Comment:
Very nice, can you add some information in the docstring about this argument? It will be displayed later on in the docs at https://gabrieldemarmiesse.github.io/python-on-whales/docker_client/#login_ecr